.github/workflows/rust.yml

```diff
@@ -2,7 +2,7 @@ name: CI
 
 on:
   push:
-    branches: [ master ]
+    branches: [ master, faidx ]
```

Comment:
This shouldn't be needed when it gets merged to master ;)

Reply:
Thanks for noting!  :)